### PR TITLE
add tom's new dyn_stack.rst lecture

### PR DESCRIPTION
**Pull request by natashawatkins**
* still having trouble with some latex labels
* need to change equation references
* add bibliography references
* reformat code to fit width

@jstac do we have any RAs to help out with these tasks? There's a lot of manual editing involved

### CODE REVIEW

**Comment by fkazemian** [comment]
- [ ]

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
screen shot 2019-02-21 at 3 21 46 pm

Also as you can see in this screenshot, some of formula doesn't have any index in the second jupyter notebook, is this on purpose?

**Comment by fkazemian** [comment]
- [ ] .

**Comment by fkazemian** [comment]
- [ ] .

**Comment by natashawatkins** [comment]
Thanks for going through this so thoroughly @fkazemian ! Yes I got rid of equation numbering where the equations weren't used, so I think that is ok. Maybe you could check that the same equations are being referenced in the text though?

Also I did edit the code to make it succincter - is there any issue with the last code block?

**Comment by fkazemian** [comment]
Thanks @natashawatkins, So I'll check the same equations for checking their references if it is alright or not.
Actually I have to install its packages and then check the result of the code, but at first I saw there is a different code and I thought there is a conflict between these two notebooks.

File: rst_files/conf.py [hunk 1]
```diff
@@ -35,11 +35,8 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [  
-	'sphinx.ext.mathjax',
```
Comment (natashawatkins) [inline rst_files/conf.py]: @fkazemian not sure we should edit this file - could you revert this change? You can keep it on your local copy

File: rst_files/dyn_stack.rst [hunk 2]
```diff
@@ -637,7 +637,8 @@ follower takes as given
 
 Thus, we let
 :math:`\check y_t' = \begin{bmatrix}\check z_t' & \check x_t'\end{bmatrix}`
-with initial condition :math:`\check z_0 = z_0` given
+with initial condition :math:`\check z_0 = z_0` given.
```
Comment (natashawatkins) [inline rst_files/dyn_stack.rst]: Sorry could we not add in full stops (.)?

Comment (fkazemian) [reply rst_files/dyn_stack.rst]: Sure I just want to be like the first notebook. I just fixed it here:
https://github.com/QuantEcon/lecture-source-py/pull/192/commits/50f89dc685209531323f1fd7ed309add447aa774

File: rst_files/conf.py (hunk 1 above)
Comment (fkazemian) [reply rst_files/conf.py]: sure at the end, I'll revert this change.

File: rst_files/dyn_stack.rst (hunk 2 above)
Comment (natashawatkins) [reply rst_files/dyn_stack.rst]: thanks, let's keep those changes

**Comment by fkazemian** [comment]
Hi @natashawatkins , I just checked its codes as well, outputs are the same as well. I also revert my changes to config.py here  https://github.com/QuantEcon/lecture-source-py/pull/192/commits/029298637601b06844918bc0964cf9e6ae7be17d

so, I change the label to checking.

**Comment by natashawatkins** [comment]
@fkazemian thanks so much for doing this. Also @mmcky said `make setup` should fix this issue

File: rst_files/conf.py (hunk 1 above)
Comment (natashawatkins) [reply rst_files/conf.py]: @fkazemian there still seems to be a remaining edit

**Comment by natashawatkins** [comment]
were all the equations ok?

**Comment by jstac** [comment]
Thanks guys!
